packages/cli/package.json

```diff
@@ -31,6 +31,7 @@
 		"@typescript-eslint/parser": "catalog:default",
 		"chalk": "^5.2.0",
 		"esbuild": "catalog:default",
+		"eslint": "catalog:default",
```

Comment:
This and a bunch of other packages didn't actually have a direct dependency on eslint, and were inheriting it from elsewhere